cli_reference/differences_oc_kubectl.adoc

```diff
@@ -0,0 +1,40 @@
+[[cli-reference-differences-oc-kubectl]]
+= Differences Between OC and Kubectl
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+A copy of `kubectl` is included with the {product-title} command line interface (CLI).
+Although there are several similarities between these two clients, this guide will
+aim to clarify the main reasons and scenarios for using one over the other.
+
+== Using OC
+
+The `oc` binary offers full support for all of the features offered by an {product-title} cluster.
```

Comment:
Stronger message will be if we write something like: 

oc binary provides the exact same capabilities as kubectl binary, on top of that it's extended to natively support {product-title} features such as: